src/components/RadioGroup/index.ts

```diff
@@ -0,0 +1,9 @@
+import RadioGroup from './RadioGroup';
+import * as CONSTANTS from './RadioGroup.constants';
+import { RadioProps as _RadioProps, GroupProps } from './RadioGroup.types';
+
+export { CONSTANTS as RADIO_GROUP_CONSTANTS };
+
+export type RadioProps = _RadioProps;
+export type RadioGroupProps = GroupProps;
```

Comment:
You can simplify the export here:
```suggestion
export * as RADIO_GROUP_CONSTANTS from './RadioGroup.constants';
export * from './RadioGroup.types';
```